app/RockTheVoteRecord.php

```diff
@@ -40,7 +40,10 @@ public function __construct($record, $config = null)
 
         if ($this->userData['mobile']) {
             // Note: Not a typo, this column name does not have the trailing question mark.
-            $this->userData['sms_status'] = str_to_boolean($record['Opt-in to Partner SMS/robocall']) ? 'active' : 'stop';
+            $smsOptIn = str_to_boolean($record['Opt-in to Partner SMS/robocall']);
+
+            $this->userData['sms_status'] = $smsOptIn ? 'active' : 'stop';
```

Comment:
Sorry, this is irrelevant to this change but just curious if there's any special reason we explicitly set a `stop` status, instead of just _not_ setting 'em to `active`?

Reply:
The reason is that we are saving their mobile number, regardless if they opt-in or not. So by not opting and setting them to `stop`, we can be sure we won't message them. I've been following the existing import logic.  There could be an option to change this and simply not save their mobile number if they didn't opt in, but the decision to save their mobile even if they don't subscribe was likely made in order to try to assign one Northstar ID per person (if I subscribe later from my phone and the RTV never captured it, I'm now in the age-old 2 Northstar users predicament)

Reply:
Ah that makes sense re the number! I assumed that we _only_ message folks who are actively `active`, and the `stop` status is a way to indicate an 'active' unsubscribe but an empty status would _also_ not get messaging.

Reply:
I'd think an empty status wouldn't get messaging too -- but since users won't be checking the opt-in box, setting to `stop` makes sense (vs null).

Reply:
Ah gotcha - so in a sense, there _is_ an active 'unsubscribe' or 'non-subscribe' happening here that we're faithfully assigning.